spec/integration/kibana/spec.yml

```
@@ -135,12 +135,25 @@ spec:
       contentMediaType: "application/json"
       pattern: '^{PACKAGE_NAME}-.+\.json$'
       forbiddenPatterns:
-        - '^.+-(ecs|ECS)\.json$' # ECS suffix is forbidden 
+        - '^.+-(ecs|ECS)\.json$' # ECS suffix is forbidden
+  - description: Folder containing security AI prompt assets
+    type: folder
+    name: "security_ai_prompt"
+    required: false
+    contents:
+    - description: A security AI prompt asset file
+      type: file
+      contentMediaType: "application/json"
+      pattern: '^{PACKAGE_NAME}-.+\.json$'
 versions:
   - before: 3.4.0
     patch:
       - op: remove
         path: "/contents/13" # remove SLO definitions
+  - before: 3.4.0
+    patch:
+      - op: remove
+        path: "/contents/6" # remove AI prompt definitions
```

Newer patches on top, so it is easier to keep track of changes.

If the new folder is added at the end, then yes, it should be `contents/14`.
```suggestion
  - before: 3.4.0
    patch:
      - op: remove
        path: "/contents/14" # remove AI prompt definitions
      - op: remove
        path: "/contents/13" # remove SLO definitions
```